xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -217,12 +221,21 @@ bool CDVDInputStreamBluray::Open()
       return false;
     }
   }
-  else
+  else if (openFiles)
```

Comment:
Get rid of the openFiles-var and just make this "else if (m_item.IsSmb())".

Reply:
It seems broken logic affects only blue-ray discs. So shouldn't this be 
```suggestion
  else if (!m_item.IsBluray())
```
instead?

Reply:
Yup, makes sense (related to my question below about NFS etc.)

Reply:
> It seems broken logic affects only blue-ray discs. So shouldn't this be
> 
> instead?

Yes that is probably better.